engine/userguide/networking/work-with-networks.md

```diff
@@ -787,6 +787,12 @@ containers connected to that network can access the alias.
 Multiple containers can share the same network-scoped alias within the same
 network. This example illustrates how this works.
 
+> **Note**: Network aliases are a legacy networking feature. Swarm services
```

Comment:
I made some edits. Maybe @johndmulhausen can work with you on further tweaks needed to get this in.